### ISSUE TITLE
Default bitDepth?

### ISSUE THREAD

**Issue by dy** [raymond-h/pcm-format#2]
Sorry for being annoying maybe, but :)

``` js
new PCMFormat({float: true}, {float: false});
```

This throws such error

``` sh
c:\Users\dmitry\Dropbox\Projects\audio-generator\node_modules\pcm-format\lib\sample.js:59
        buffer['writeUInt' + bitDepth + byteOrder](val, 0);
                                                  ^
TypeError: Property 'writeUIntundefinedLE' of object  is not a function
```

That is because if bitDepth is `undefined`, it can’t guess it. Maybe for default case there is a sense to use 16 bit as a default value?

**Comment by raymond-h** [comment]
No worries, you're not being annoying :)

Considering there are such a variety of bit depths used, it didn't really seem to make sense to me to have a default bit depth if not using floating-point samples. I have seen several cases of signed 16 bit depth as default, however, so perhaps YMMV.

**Comment by dy** [comment]
Well, the reason behind using default value is just let users of the package type less to get started, simply usability. (according to the priority chain _users over authors over implementors over specifiers over theoretical purity_). And 16 bits is sane as `node-speaker` uses that by default, and possibly it is the most common case.

**Comment by raymond-h** [comment]
Of course, default values are good - it just didn't seem like it would be all that useful to actually have a default value for bit depth. But since there are a lot of modules using 16 as their default bit depth, sure, this module can do that too.

**Comment by dy** [comment]
Yes, all the usefullness is just in hypothetical

``` js
new PCMFormat({float: true});
```

instead of

``` js
new PCMFormat({float: true}, {float: false, bitDepth: 16, /*...what else is there? */});
```

Just to let users point out the input format and convert to the standard format _16 bits int_, unless it is directly pointed as something specific. Just not to force them look up for the docs each time they want to use that module :)

**Comment by dy** [comment]
@raymond-h so should I fork it?

**Comment by raymond-h** [comment]
Sure, go ahead.
